setup.py

```diff
@@ -12,8 +13,11 @@
 requires = [
     'requests>=1.0.0,<3.0',
     'pycountry',
-    'enum34',
 ]
+# Incompatible with Py 3.5+: https://bitbucket.org/stoneleaf/enum34/issues/5
+if (sys.version_info[0] < 3 or
+        sys.version_info[0] == 3 and sys.version_info[1] < 5):
+    requires.append('enum34')
```

Comment:
If you're pedantic this should be `< 4` and not `< 5`. The package is called `enum34` because it back-ports enums from  Python 3.4, so you don't need to install it for 3.4 either.